### PR TITLE
Updates for U10

### PR DESCRIPTION
**Pull request by nCastle1**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Release notes
 
+## 1.0.9
+
+- Removes support for iOS 12.
```
Comment (esreli) [inline RELEASE.md:5]:
```suggestion
- Deprecate support for iOS 12.
```

**Review by esreli — CHANGES_REQUESTED** [review]
This PR generally looks good. I'm trusting the app version has been updated to `1.0.9` in all locations, ditto update min SDK Android (23) & iOS (13.0). I have one general question.
What are all of the changes contained by `src/MapsApp.Android/Resources/Resource.Designer.cs`? Were these manual or system derived changes?

---

With the rollout of the new site, there are a number of broken links.

### `README`

- Line `77` - `[new application](https://developers.arcgis.com/applications/new)`
- Line `125` - `[License your app](https://developers.arcgis.com/net/latest/wpf/guide/license-your-app.htm)`

### `docs/README`

- Section: Place search & geocoding - `[here](https://developers.arcgis.com/net/latest/wpf/guide/loadable-pattern.htm)`
- Section: Place search & geocoding - `[Geocoding](https://developers.arcgis.com/net/latest/wpf/guide/search-for-places-geocoding-.htm)`
- Section: Authentication - `[Go here](https://developers.arcgis.com/net/latest/wpf/guide/use-the-authentication-manager.htm)`

**Comment by nCastle1** [comment]
Resources.Designer.cs is a generated file.

**Comment by nCastle1** [comment]
App has been fully tested on all platforms, everything working as expected.

File: README.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ If there are changes made in the Original repository, you can sync the fork to k
 Maps app uses authenticated ArcGIS Online services for routing and geocoding. You need to configure the app for OAuth authentication.
 
 1. Log in to [developers.arcgis.com](https://developers.arcgis.com).
-2. Create a [new application](https://developers.arcgis.com/applications/new).
+2. Create a [new application](https://developers.arcgis.com/applications).
```
Comment (esreli) [inline README.md:77]:
```suggestion
2. Create a [new application](https://developers.arcgis.com/applications/).
```

File: README.md [hunk 3]
```diff
@@ -122,4 +122,4 @@ Unless required by applicable law or agreed to in writing, software distributed
 
 A copy of the license is available in the repository's [LICENSE](https://github.com/Esri/maps-app-dotnet/blob/master/LICENSE) file.
 
-For information about licensing your deployed app, see [License your app](https://developers.arcgis.com/net/latest/wpf/guide/license-your-app.htm).
+For information about licensing your deployed app, see [License your app](https://developers.arcgis.com/applications).
```
Comment (esreli) [inline README.md:125]:
```suggestion
For information about licensing your deployed app, see [License your app](https://developers.arcgis.com/ios/license-and-deployment/license/).
```

**Review by esreli — APPROVED** [review]
One link fix and one pedantic link fix ;) LGTM!

**Pull request merged**